apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/Ethereum1559CustomFees.tsx

```diff
@@ -65,14 +73,32 @@ const Ethereum1559CustomFees = ({
         maxPriorityFeePerGas,
       },
     }));
+
   const { errors, warnings } = status;
   const { maxPriorityFee: maxPriorityFeeError, maxFee: maxFeeError } = errors;
-  const { maxPriorityFee: maxPriorityFeeWarning, maxFee: maxFeeWarning } =
-    warnings;
+  const { maxFee: maxFeeWarning } = warnings;
+  let { maxPriorityFee: maxPriorityFeeWarning } = warnings;
+
+  // give user a warning if maxFeePerGas is lower than pending transaction maxFeePerGas + 10% of pending transaction maxPriorityFeePerGas for edit eth transaction feature
+  if (!maxPriorityFeeWarning && transactionRaw?.maxPriorityFeePerGas) {
+    const maxPriorityFeeGap: number = getEnv(
+      "EDIT_TX_EIP1559_MAXPRIORITYFEE_GAP_SPEEDUP_FACTOR",
+    );
+    const lowerLimitMaxFeePerGas = new BigNumber(
+      transactionRaw.maxFeePerGas || 0,
+    ).plus(
+      new BigNumber(transactionRaw.maxPriorityFeePerGas).times(
+        maxPriorityFeeGap,
+      ),
+    );
+    if (transaction?.maxFeePerGas?.isLessThan(lowerLimitMaxFeePerGas)) {
+      maxPriorityFeeWarning = new MaxFeeTooLow();
```

Comment:
I guess it is mistake?
1. Shouldn't it be "maxFeeWarning = new MaxFeeTooLow();" ?
2. I think you should also check the maxPriorityFee to ensure that it is at least 10% higher than before?
3. Are them an error or just a warning ? I guess we should prevent users from continuing the flow